cves/kernel/CVE-2013-1763.yml

```diff
@@ -465,13 +489,11 @@ CWE_instructions: |
     CWE: [123, 456]     # also ok
     CWE: 123            # also ok
 CWE:
-- 20
-CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+  - 20
+CWE_note: 'Manually confirmed.'
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
-CVSS:
+nickname: Batman vs. Superman
+CVSS: 2.0 (AV:L/AC:L/Au:N/C:P/I:P/A:C)
```

Comment:
https://nvd.nist.gov/vuln/detail/CVE-2013-1763 says that the cvss should have C:C, not C:P